src/StoreApi/SessionHandler.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Automattic\WooCommerce\StoreApi;
+
+use Automattic\Jetpack\Constants;
+use Automattic\WooCommerce\StoreApi\Utilities\JsonWebToken;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * SessionHandler class
+ */
+final class SessionHandler extends \WC_Session {
+	/**
+	 * Token from HTTP headers.
+	 *
+	 * @var string
+	 */
+	protected $token;
+
+	/**
+	 * Table name for session data.
+	 *
+	 * @var string Custom session table name
+	 */
+	protected $_table; // phpcs:ignore PSR2.Classes.PropertyDeclaration.Underscore
+
+	/**
+	 * Expiration timestamp.
+	 *
+	 * @var int
+	 */
+	protected $_session_expiration; // phpcs:ignore PSR2.Classes.PropertyDeclaration.Underscore
```

Comment:
This prefix has to stay because it's used in WC_Session. We don't need to redeclare it but maybe this was done to prevent linting errors?

Reply:
It is used on WC_Session_Handler, not the abstract WC_Session, we need to declare it. What we don't need is keeping up with the underscore prefix like the $_table example. I'll remove that for those, as they were only done to keep WC_Session_Handler "tradition" 👍🏼